html/learn.html

```diff
@@ -201,6 +202,36 @@ <h2 id="everlastingCollection">Everlasting Collection</h2>
       </p>
     </div>
 
+    <div class="feature-block">
+      <h2 id="editingNotepad">Editing Notepad</h2>
+    
+      <p>A Notepad that you can type notes into while editing releases. These notes will be saved across different
+        editing and browser sessions.</p>
+
+      <p>The Notepad can be expanded by clicking the flyout tab on the right side of the screen while editing a release,
+        master release, artist, label, or images.</p>
+      
+      <img src="../img/learn/editing-notepad-tab.jpg" />
+      
+      <p>From here you can click the "Edit" (✎) button to edit your Notepad...</p>
+      
+      <img src="../img/learn/editing-notepad-edit.jpg" />
+      
+      <p>and then "Save" (✔) it using the same button</p>
+      
+      <img src="../img/learn/editing-notepad-save.jpg" />
+      
+      <p>If you paste in a URL, it will be auto-formatted for you and turned into a clickable hyperlink, so that you can click
+        on it without needing to manually copy/paste.</p>
+      
+      <img src="../img/learn/editing-notepad-hyperlink.jpg" />
+      
+      <p><em>Note:</em> The notes in the Notepad will be cleared if you clear your Discogs data in your browser, as it is
+        stored in
+        <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage">Local Storage</a>. It will also not
+        be saved if you browse using Private/Incognito mode and then close your browser.</p>
+    </div>
+
```

Comment:
This block should be bumped up one section and go right between `Demand Index` and `Everlasting Collection` 👍